### PR TITLE
doc: update contributing.md

### PR DESCRIPTION
**Pull request by YTGhost**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

#2187 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,15 +11,22 @@ Please do check our [Code of Conduct](CODE_OF_CONDUCT.md) before making contribu
...
 * [Reporting general issues](#reporting-general-issues)
 * [Code and doc contribution](#code-and-doc-contribution)
+* [Review](#review)
 * [Engage to help anything](#engage-to-help-anything)
+* [At Last](#at-last)
 
 ## Reporting security issues
 
-We take security issues seriously and discourage anyone to spread security issues. If you find a security issue in OpenYurt, please do not discuss it in public and even do not open a public issue. Instead we encourage you to send us a private email to  [openyurt@gmail.com](mailto:openyurt@gmail.com) to report the security issue.
+We take security issues seriously and discourage anyone to spread security issues. If you find a security issue in OpenYurt, please do not discuss it in public and even do not open a public issue. Instead we encourage you to send us a private email to [kubernetes-security@service.aliyun.com](mailto:kubernetes-security@service.aliyun.com) to report the security issue.
```
Comment (rambohe-ch) [inline CONTRIBUTING.md:20]: kubernetes-security@service.aliyun.com --> security@mail.openyurt.io

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -123,9 +131,32 @@ Commit content represents all content changes included in one commit. We had bet
 * Avoid very large change in a commit;
 * Be complete and reviewable for each commit.
 
-### PR Description
+### PR Guidelines
+
+PR is the only way to make change to OpenYurt project. Before submitting a Pull Request, you should check your local git repository and keep pace with the OpenYurt repo to avoid the merge conflict. In addition, you should have some knowledge of how does the OpenYurt [CI Workflow](./ci-workflow.md) work.
```
Comment (rambohe-ch) [inline CONTRIBUTING.md:136]: `./ci-workflow.md` is a dead link.

File: CONTRIBUTING.md (hunk 1 above)
Comment (YTGhost) [reply CONTRIBUTING.md]: > [kubernetes-security@service.aliyun.com](mailto:kubernetes-security@service.aliyun.com) --> [security@mail.openyurt.io](mailto:security@mail.openyurt.io)

Should the email address in this [document](https://github.com/openyurtio/openyurt/blob/master/SECURITY.md) also be changed?

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**